docs/source/install/index.rst

```diff
@@ -3,15 +3,15 @@ Installation
 
 For a quick evaluation of |st2| and package-based installation, start from
 :doc:`Overview <./overview>`, and proceed with the guide for your Linux distribution.
-
+Please note that only 64 bit architecture is currently supported.
 
 .. rubric:: Installations
 
 .. toctree::
     :maxdepth: 1
 
     Reference Deployment <overview>
-    Ubuntu / Debian <deb>
+    Ubuntu / Debian 14.04 (Trusty) <deb>
```

Comment:
This line just doesn't look nice now :)

BTW, for those who is OK not to get support, there are debian `wheezy` and `jessie` packages in unstable repo. I know some people are using it.
What do you think about the future of those packages?